roles/ci_lvms_storage/README.md

```diff
@@ -0,0 +1,153 @@
+# ci_lvms_storage
+
+The `ci_lvms_storage` role aims to be a drop in replacement
+for `ci_local_storage`. It uses LVMS (Logical Volume Manager
+Storage) based on the TopoLVM CSI driver to dynamically
+provision local storage built from block devices on OCP
+nodes.
+
+This role requires `cifmw_lvms_disk_list` to contain a list of
+paths to block devices which already exist on OCP nodes. If
+the `cifmw_devscripts_config_overrides.vm_extradisks_list` list
+contains `"vda vdb"`, then the `devscripts` role will create these
+block devices and `cifmw_lvms_disk_list` should be set to
+`['/dev/vda', '/dev/vdb']`. This role will then pass the disk list
+to the `deviceSelector` list used by the `LVMCluster` CRD.
+
+## Privilege escalation
+
+It does not have any tasks which use become, but it does need to use
+`roles/ci_lvms_storage/templates/lvms-namespace.yaml.j2` to create
+create an OpenShift namespace where it then sets up a subscription
+as per `roles/ci_lvms_storage/templates/subscription.yaml.j2`. It
+uses the `cifmw_lvms_disk_list` list to find disks which it then wipes
+clean and adds to an LVMS cluster.
+
+## Parameters
+
+### ci-framework parameters
+
+* `cifmw_lvms`: (Boolean) Whether or not to use LVMS (default: `false`)
```

Comment:
This should be added to `docs/source/usage/01_usage.md`

Reply:
[Suggestion] Rather than using a bool for this behavior it would be clearer to use  something like one of these suggestions:
- cifmw_storage_class: "lvms-local-storage" | cifmw_storage_class: "local-storage"
- cifmw_storage_class_provisioner: "ci_lvms_storage" | cifmw_storage_class_provisioner: "ci_local_storage"

Reply:
> This should be added to `docs/source/usage/01_usage.md`

done.

Reply:
> [Suggestion] Rather than using a bool for this behavior it would be clearer to use something like one of these suggestions:
> 
> * cifmw_storage_class: "lvms-local-storage" | cifmw_storage_class: "local-storage"
> * cifmw_storage_class_provisioner: "ci_lvms_storage" | cifmw_storage_class_provisioner: "ci_local_storage"

It's the LVMS operator which is prepending the prefix. I.e. a prefix of `"lvms-"` is prepended on whatever variable I pass for `cifmw_storage_class` by the LVMs operator. So I default the variable to `local-storage` and then it creates a storage class called `lvms-local-storage`.

  https://github.com/fultonj/ci-framework/blob/ci_lvms_storage/roles/ci_lvms_storage/defaults/main.yml#L26

I do want a single boolean to make it easy for people to use the new feature or not.